ReverseIndexManip/AbstractReverseIndexManip.py

```diff
@@ -104,16 +122,10 @@ def endOfIndexes(self):
 				return False
 		return True
 
-	def correspondingFile(self, number):
-		"""
-		Returns the corresponding name of the file number
-		"""
-		assert(number < len(self.documents))
-		return self.documents[number]
-
-	@classmethod
-	def merge(cls, indexesList, indexDestinationDirectory):
+	def write(self, key, documentList, valueList):
 		"""
-		Merges two Reverse indexes
+		Writes in the document the key and associated document and value
```

Comment:
The parameter is "valueList" but you are saying "value" in your comment. Is it an array of value or a sigle one there ?